faithcomesbyhearing/dbp
# Task-2284 API: fix order timestamp endpoint

# Description
Added the feature to sort by `verse_sequence` when the fileset is either an audio_stream or audio_drama_stream.

## Issue Link
[Bug 2284](https://dev.azure.com/keyholesoftware/FCBH%20-%20Bible%20Brain/_workitems/edit/2284): API: fix order timestamp endpoint

## How Do I QA This
You should run the following postman test and it has to pass.
https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/request/12519377-571516be-b308-4083-a631-65ef7078481b?action=share&source=copy-link&creator=17122915&active-environment=810fd94b-9392-43e2-9f13-943e8d323135
